packages/nimble-components/package.json

```diff
@@ -114,9 +114,12 @@
     "prettier": "^2.0.0",
     "prettier-eslint": "^15.0.1",
     "prettier-eslint-cli": "^7.1.0",
+    "puppeteer": "^10.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Are these required? I expect not, we don't import them anywhere, they can just be handled as peer deps

Reply:
Nope! They're gone 👋